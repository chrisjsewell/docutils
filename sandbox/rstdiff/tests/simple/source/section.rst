First section
=============

This is text in the first section.

.. class:: change-added

Section two
===========

This was missing.

Third section
=============

Text in third section.

Anchor
======

Don't reuse words!

.. class:: change-removed

Obsolete part
=============

Obsolete text.

Fourth section
==============

Number 4!

.. class:: change-replaced

Replacable section
==================

Replacable text.

.. class:: change-replacement

Section replacement
===================

Text replacement.
