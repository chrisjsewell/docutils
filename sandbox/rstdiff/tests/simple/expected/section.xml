<?xml version="1.0" encoding="utf-8"?>
<!-- Generated by Docutils 0.6 -->
<document source="source/section.rst">
    <section ids="first-section" names="first\ section">
        <title>
            First section
        </title>
        <paragraph>
            This is text in the first section.
        </paragraph>
    </section>
    <section classes="change-added" ids="second-section" names="second\ section">
        <title>
            Second section
        </title>
        <paragraph>
            This was missing.
        </paragraph>
    </section>
    <section ids="third-section" names="third\ section">
        <title>
            Third section
        </title>
        <paragraph>
            Text in third section.
        </paragraph>
    </section>
    <section classes="change-deleted" ids="obsolete-section" names="obsolete\ section">
        <title>
            Obsolete section
        </title>
        <paragraph>
            Obsolete text.
        </paragraph>
    </section>
    <section ids="fourth-section" names="fourth\ section">
        <title>
            Fourth section
        </title>
        <paragraph>
            Number 4!
        </paragraph>
    </section>
    <section classes="change-replaced" ids="replacable-section" names="replacable\ section">
        <title>
            Replacable section
        </title>
        <paragraph>
            Replacable text.
        </paragraph>
    </section>
    <section classes="change-replacement" ids="section-replacement" names="section\ replacement">
        <title>
            Section replacement
        </title>
        <paragraph>
            Text replacement.
        </paragraph>
    </section>
</document>
